### ISSUE TITLE
Filter out internal and test users - with multiple values does not work

### ISSUE THREAD

**Issue by SimonHausdorf** [PostHog/posthog#12523]
## Bug description

As described here https://posthog.com/tutorials/filter-internal-users#step-3-add-more-filters multiple values should be separated by comma.

## How to reproduce

1. Add the following filter to filter out internal and test users: `WHERE host DOESN'T CONTAIN localhost,vercel,otherDomain`
2. Insights with filter activated still shows results with the listed hosts from the filter

## Environment

- [ ] PostHog Cloud
- [x] self-hosted PostHog, version/commit: latest

## Additional context

I believe that the issue is with `doesn't contain` and that it would work on `doesn't equal` but I did not test it.

**Comment by timgl** [comment]
It's not comma separated, you should be able to separately add each value like this: 
<img width="740" alt="image" src="https://user-images.githubusercontent.com/1727427/198995732-ebd51457-99f8-4a08-98bf-8e2f36f0afaf.png">

**Comment by SimonVillage** [comment]
That sadly does only work with `doesn't equal` but not with `doesn't contain`

**Comment by timgl** [comment]
Ah apologies, I read over that bit. You're right, we currently don't support multiple values with contains/does not contain. You can just have multiple filters though as a work around.

**Comment by SimonVillage** [comment]
That workaround also does not really work as the settings page only allows `AND` filters not `OR` which would be required.

**Comment by timgl** [comment]
In this case you can do `does not contain localhost AND does not contain vercel` right?

**Comment by SimonVillage** [comment]
Right, that should work. I got confused because of `should not contain` and `should contain`.

My initial thought was that "filter out internal and test users" should have a `should contain` filter. That was discussed here already: https://github.com/PostHog/posthog/issues/9483

I got it working on my end now but will leave the issue open.

**Comment by timgl** [comment]
Perfect. Yep we'll leave this open for the multiple option thing
